config/clusters/2i2c/cluster.yaml

```diff
@@ -47,7 +47,8 @@ hubs:
       # matching value for jupyterhub.custom.2i2c.add_staff_user_ids_of_type!
       enabled: false
     helm_chart_values_files:
-      - enc-binder.secret.values.yaml
+      - binder-staging.values.yaml
```

Comment:
```suggestion
```

This file doesn't exist anymore - _unless_ you want to migrate what you've removed from the helm chart into that file?

Reply:
@sgibson91 whoops, forgot to `git add`! Perils of relying so heavily on `git commit -p`... Added now!